week1/homework/out/Contact.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+exports.Contact = class Contact {
```

Comment:
you are only exporting one thing in this file, so no need to export Contact under the property Contact. You can just write `exports = class Contact { ... }` instead.

Then you can change:
```js
var Contact = require('./Contact.js');
let rafah = new Contact.Contact("Rafah", 45679908);
```

to

```js
var Contact = require('./Contact.js');
let rafah = new Contact("Rafah", 45679908);
```